src/bitcoin-cli.cpp

```diff
@@ -39,6 +39,7 @@ UrlDecodeFn* const URL_DECODE = urlDecode;
 
 static const char DEFAULT_RPCCONNECT[] = "127.0.0.1";
 static const int DEFAULT_HTTP_CLIENT_TIMEOUT=900;
+static const int DEFAULT_WAIT_CLIENT_TIMEOUT=0;
```

Comment:
could be `constexpr`

Reply:
Had to catch up about `constexpr`, but good point :+1: